hieradata/role/jenkins-slave.yaml

```diff
@@ -0,0 +1,139 @@
+    type: 'dword'
+
+psick::puppet::pe_agent::manage_environment: true
+psick::windows::features::use_defaults: true
+psick::windows::features::install:
+  'file-services':
+    ensure: present
+  'storage-services':
+    ensure: present
+  'net-framework-45-core':
+    ensure: present
+  'was':
+    ensure: present
+  'was-net-environment':
+    ensure: present
+  'was-config-apis':
+    ensure: present
+
+psick::windows::services::use_defaults: true
+psick::windows::services::managed:
+  'RemoteRegistry':
+    ensure: 'stopped'
+    enable: 'manual'
+  'Spooler':
+    ensure: 'running'
+    enable: 'true'
+  'TCPIP6':
+    ensure: 'stopped'
+    enable: 'manual'
+
+# Windows firewall settings.
+windows_firewall::profile_state: 'on'
+windows_firewall::in_policy: 'AllowInbound'
+# FIXME: Why commented? Do we need it?
+# windows_firewall::in_policy: 'BlockInbound'
+windows_firewall::out_policy: 'AllowOutbound'
+windows_firewall::rule_key: 'windows_networks'
+# Purge unmanaged firewall rules
+windows_firewall::purge_rules: false
+
+# Testing users
+psick::users::users_hash:
+  antony:
+    ensure: present
+    comment: 'Antony Gelberg'
+  alessandro:
+    ensure: present
+    comment: 'Alessandro Francheschi'
+  shachar:
+    ensure: present
+    comment: 'Shachar Shapira'
+
+# Sample rules
+windows_networks:
+  Remote Desktop - User Mode (TCP-In):
+    description: Inbound rule for the Remote Desktop service to allow RDP traffic.
+      [TCP 3389]
+    application_name: C:\Windows\system32\svchost.exe
+    service_name: termservice
+    protocol: 6
+    local_ports: '3389'
+    remote_ports: "*"
+    local_addresses: "*"
+    remote_addresses: "*"
+    direction: 1
+    interface_types: All
+    enabled: true
+    grouping: "@FirewallAPI.dll,-28752"
+    profiles: 7
+    edge_traversal: false
+    action: 1
+    edge_traversal_options: 0
+  Remote Desktop - User Mode (UDP-In):
+    description: Inbound rule for the Remote Desktop service to allow RDP traffic.
+      [UDP 3389]
+    application_name: C:\Windows\system32\svchost.exe
+    service_name: termservice
+    protocol: 6
+    local_ports: '33891'
```

Comment:
This looks like a type (33891 vs 3389). So I guess it doesn't work, and it's not necessary?